zerver/openapi/zulip.yaml

```diff
@@ -282,6 +282,41 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/JsonSuccess'
+  /attachments:
+    get:
+      description: Get all the attachments of the requesting user.
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/JsonSuccess'
+                - properties:
+                    attachments:
+                      type: array
+                      description: A list of `attachment` objects, each containing
+                        details about an attachment belonging to the user.
+                    upload_space_used:
+                      type: integer
+                      description: Used upload space by the user in bytes.
+                - example:
+                      {
+                        "result": "success",
+                        "msg": "",
+                        "attachments": [
+                          {
+                            "id": 1,
+                            "name": "test.txt",
+                            "path_id": "foo/bar/test.txt",
+                            "size": None,
+                            "create_time": 1584650973000,
+                            "messages": []
```

Comment:
We should probably have an example that's been sent in a message so `messages` is present here.